spyderlib/widgets/ipython.py

```diff
@@ -682,9 +682,9 @@ def shellwidget_config(self):
         spy_cfg.IPythonWidget.kind = 'rich'
         
         # Gui completion widget
-        gui_comp_o = self.get_option('use_gui_completion')
-        completions = {True: 'droplist', False: 'ncurses'}
-        spy_cfg.IPythonWidget.gui_completion = completions[gui_comp_o]
+        completion_widget_o = CONF.get('ipython_console', 'completion_widget')
+        completions = {0: "droplist", 1: "ncurses", 2: "plain"}

```

Comment:
Note that these values remain the same as IPython ends up validating them against an Enum traitlet so the renaming only happens at the spyder/combobox level.